backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestEvent.java

```diff
@@ -156,20 +172,10 @@ public DeviceSpecimenType getDeviceSpecimenType() {
   // getResultSet()
   public TestResult getTestResult() {
     final String COVID_LOINC = "96741-4";
-    if (this.results != null) {
-      Optional<Result> resultObject =
-          this.results.stream()
-              .filter(result -> COVID_LOINC.equals(result.getDisease().getLoinc()))
-              .findFirst();
-      if (resultObject.isPresent()) {
-        return resultObject.get().getTestResult();
-      }
-    }
-    return super.getResult();
-  }
-
-  @Override
-  public TestResult getResult() {
-    return getTestResult();
```

Comment:
I'm wondering if we should update the name of this method to be `getCovidTestResult` so we can be more clear about what result we are getting back?

Reply:
Thank you so much for doing this -- it is so much easier follow!! 😸